desc/backend.py

```diff
@@ -5,6 +5,7 @@
 import warnings
 
 import numpy as np
+import sympy as sp  # noqa: F401
```

Comment:
i don't think this needs to be in backend, is there any reason we don't just import it normally where its needed?

Reply:
I had moved it just to be in backend but I can move it back. Just thought I remembered someone saying we should import stuff like this in backend.